### ISSUE TITLE
CelebA-HQ inpainting

### ISSUE THREAD

**Issue by sobieskibj** [Hammour-steak/GOUB#13]
Hi,
congratulations on a very nice paper. Where can I find specific details about the CelebA-HQ inpainting task? What were the training and validation splits? Where can I find the masks used during training?

**Comment by Hammour-steak** [comment]
In the latest version of the README, we have included the [dataset](https://github.com/Hammour-steak/GOUB?tab=readme-ov-file#test). Specifically, the training set contains 27,000 images, and the validation set contains 3,000 images, which is consistent with many other papers. The masks are sourced from [here](https://github.com/Algolzw/image-restoration-sde/tree/main/codes/config/inpainting/data/datasets/gt_keep_masks). The inference follows a masking order from 1 to 100.

**Comment by sobieskibj** [comment]
Thank you very much for the quick and detailed response :)
